_redirects

```diff
@@ -0,0 +1,3 @@
+# See:  https://docs.netlify.com/routing/redirects/
+#       https://docs.netlify.com/routing/redirects/redirect-options/
+/schemas/latest/* /schemas/0.5/:splat    200
```

Comment:
shouldnt schemas/latest point to the chemas/0.6-alpha? just checking

Reply:
Well, I had this question as well, but `latest` to me refers to the latest released version, not to the master branch. What do you think?

Reply:
maybe snapshot for chemas/0.6-alpha then? wdyt?

Reply:
or latest/snapshot
latest/release ?

Reply:
Cool! I like the snapshot thing. So we can have:

- `latest` -> latest stable version like `0.5`
- `snapshot` -> latest `snapshot` version like `0.6-alpha`

Or `release` instead of `latest`? I see more `latest` than `release`. So I honestly don't know. Both works for me.

Reply:
Or `stable`?

Reply:
I think `latest` is what is used the most for schemas.

Reply:
Aaahhharg! I don't know honestly! hahahah

Reply:
hehe idk either. pick one and lets go w/ it.